doc/migration_guide.md

```diff
@@ -0,0 +1,15 @@
+# Migration Guide
+A collection of tips for migrating away from deprecated features.
+
+## Deprecation of `FixedLenStringArray`.
+The issue with `FixedLenStringArray` is that it is unable to avoid copies.
+Essentially, this class acts as a means to create a copy of the data in a
+format suitable for writing fixed-length strings; and the class acts as a tag
+for HighFive to overload on. The support of `std::string` in HighFive. Since
```

Comment:
rewrite a bit here?

This small sentence seems awkward.

Reply:
Very, thanks for noticing. I've tried again.